### DISCUSSION TITLE
Model editor : empty snippet

### DISCUSSION THREAD

**Discussion by clementPigeron** [google/model-viewer#2767] [Q&A]
Hello everyone !

I use the model editor (https://modelviewer.dev/editor/) to upload my glb files, edit the scene and download a zip to integrate in my website (to have a model viewer directly on it).
I recently had a new issue : the zip files doesn't work on my website anymore and I noticed that when I click on "download scene", I get a zip file with an empty snippet.
I tried replacing the empty snippet with one that I know works, but no result... I get this issue with any glb I upload or take from the library.

You'll get from this that I'm not really familiar with coding nor github, sorry for this.

Thanks !

**Comment by elalish** [comment]
I've been refactoring the editor lately and it definitely looks like I broke something. Thanks for the report! Hopefully you can go back to a working version of your website and just copy-paste the snippet from the editor as a  work around.

**Reply by clementPigeron** [reply]
Thanks for your reply ! 
Yes in the meantime I managed to replace the files in my already working projects to make it work somehow.
I hope the issue isn't serious
